examples/gw150914/PyCBCInspiral.ipynb

```diff
@@ -909,14 +909,14 @@
    "source": [
     "h1_triggers = pycbc.io.hdf.SingleDetTriggers(\n",
     "    'H1-INSPIRAL_FULL_DATA_JOB0-1126257771-1837.hdf',\n",
-    "    'H1L1-GW150914_BANK-1126051217-3331800.hdf',\n",
-    "    None, None, None, 'H1'\n",
+    "    'H1'\n",
+    "    'bank_file=H1L1-GW150914_BANK-1126051217-3331800.hdf',\n",
     ")\n",
     "\n",
     "l1_triggers = pycbc.io.hdf.SingleDetTriggers(\n",
     "    'L1-INSPIRAL_FULL_DATA_JOB0-1126258302-1591.hdf',\n",
-    "    'H1L1-GW150914_BANK-1126051217-3331800.hdf',\n",
-    "    None, None, None, 'L1'\n",
+    "    'L1'\n",
+    "    'bank_file=H1L1-GW150914_BANK-1126051217-3331800.hdf',\n",
```

Comment:
I'm not actually sure this will work to directly change this like this

Reply:
I had a look and it was wrong - however the notebook instructions are to use a specific version of pycbc (v1.7.5), and the notebook doesn't work with the current master for other reasons.

So I think it may be best for me to completely remove any changes to this file and not worry about it